radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/application/LocalMempoolPeriodicSubmittor.java

```diff
@@ -57,11 +61,19 @@ private void act(RunningNetwork network, Command command, BFTNode node) {
 		return commands.observeOn(Schedulers.io());
 	}
 
-	public void run(RunningNetwork network) {
+	@Override
+	public void start(RunningNetwork network) {
+		this.isRunning = true;
 		Observable.interval(1, 10, TimeUnit.SECONDS)
```

Comment:
Would rather see this implemented via the disposable returned from `.subscribe(commands);`